### DISCUSSION TITLE
Visual studio auto-completion does not work when Tokens is set.

### DISCUSSION THREAD

**Discussion by picoArts** [chakra-ui/chakra-ui#8995] [Q&A] [answered]
If I set Tokens and execute the following command, auto-completion for color and size will not work.

> npx @chakra-ui/cli typegen ./src/lib/chakra-ui/theme.ts

I removed node_modules and installed it again, autocompletion is back only with default settings

version
@chakura-ui/react:3.0.0
@chakura-ui/cli:3.0.0

↓theme.ts
```
import { createSystem, defaultConfig } from "@chakra-ui/react";
 
export const system = createSystem(defaultConfig, {
  strictTokens: true,
  theme: {
    semanticTokens: {
      colors: {
        danger: { value: "red" },
      },
    },
  },
});
```

**Comment by segunadebayo** [comment, marked as answer]
- Try restarting your TS server if it doesn't pick it up
- Install the @chakra-ui/cli locally as a devDeps

**Reply by picoArts** [reply]
Unfortunately I have already tried that.
It did not work.
